bundles/org.openhab.binding.bambulab/src/main/java/org/openhab/binding/bambulab/internal/TrayHelper.java

```diff
@@ -14,6 +14,7 @@
 
 import static java.lang.Integer.parseInt;
 import static org.openhab.binding.bambulab.internal.BambuLabBindingConstants.AmsChannel.*;
+import static org.openhab.binding.bambulab.internal.BambuLabBindingConstants.AmsChannel.TrayId.MAX_AMS_TRAYS;
```

Comment:
Unexpected, as the wildcard should already handle this.

Reply:
idk but this import has to stay
![obraz](https://github.com/user-attachments/assets/cecb364b-dc42-4ca6-8bde-b2231e7e2d13)

Reply:
Hmpfz. Okay.